content/200-concepts/100-components/02-prisma-client/050-filtering-and-sorting.mdx

```diff
@@ -234,6 +234,36 @@ const users = await prisma.user.findMany({
 
 See also: [Case sensitivity](case-sensitivity)
 
+### Filtering FAQs
+
+#### How does filtering work at the database level?
+
+For MySQL and PostgreSQL, Prisma Client utilizes the [`LIKE`](https://www.w3schools.com/sql/sql_like.asp) (and [`ILIKE`](https://www.postgresql.org/docs/current/functions-matching.html#FUNCTIONS-LIKE)) operator to search for a given pattern. The operators have built-in pattern matching using symbols unique to `LIKE`. The pattern-matching symbols include `%` for zero or more characters (similar to `*` in other regex implementations) and `_` for one character (similar to `.`)
+
+To match the literal characters, `%` or `_`, make sure you escape those characters. For example:
+
+```ts
+const users = await prisma.user.findMany({
+  where: {
+    name: {
+      startsWith: '_benny',
+    },
+  },
+})
+```
+
+The above query will match any user whose name starts with a character followed by `benny` such as `7benny` or `&benny`. If you instead wanted to find any user whose name starts with the literal string `_benny`, you could do:
+
+```ts highlight=4
+const users = await prisma.user.findMany({
+  where: {
+    name: {
+      startsWith: '\\_benny', // note that the `_` character is escaped
```

Comment:
For whatever reason my local machine is stripping out `\`. adding `\\` is not working. 😠 
```suggestion
      startsWith: '\_benny', // note that the `_` character is escaped
```

Reply:
Would escaping the second / with \ work? 🤔

Reply:
the `//` isn't the problem, it's `'\_benny'` which some sort of git commit hook transfers to `'_benny`. 

Committing this via GH UI seems to work though

Reply:
Would another commit (from the same or different PR) reset this? That might be worth considering. I would also recommend internally raising this in `#website`/ `#docs` channels.

Reply:
That is probably via https://github.com/prisma/docs/blob/main/.husky/pre-commit

Reply:
Fixed! Thank you @nilubava